commit 1e1350fae0b20d01e17da6f91b555e979958cded
Author: Patrick Mooney <pmooney@pfmooney.com>
Date:   2019-04-26T22:03:15+00:00 (5 months ago)
    
    OS-7773 remove comma from copyright pbchk
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/tools/onbld/Checks/Copyright.py b/usr/src/tools/onbld/Checks/Copyright.py
index 7727265622..00ca8829b9 100644
--- a/usr/src/tools/onbld/Checks/Copyright.py
+++ b/usr/src/tools/onbld/Checks/Copyright.py
@@ -25,7 +25,7 @@
 #
 
 # Copyright 2008, 2010, Richard Lowe
-# Copyright 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 
 # Make sure there is a copyright claim for the current year.
 
@@ -38,7 +38,7 @@ def is_copyright(line):
 	return re.search(r'Copyright (?!\[yyyy\])', line)
 
 def is_current_copyright(line):
-	return re.search(r'Copyright %s, Joyent, Inc.$' % time.strftime('%Y'), line)
+	return re.search(r'Copyright %s Joyent, Inc.$' % time.strftime('%Y'), line)
 
 def copyright(fh, filename=None, output=sys.stderr):
 	ret = rights = goodrights = 0
